### DISCUSSION TITLE
Deep R Programming

### DISCUSSION THREAD

**Discussion by chartgerink** [epiverse-trace/epiverse-trace.github.io#280] [Resources]
### Context

> Deep R Programming is a comprehensive and in-depth introductory course on one of the most popular languages for data science. It equips ambitious students, professionals, and researchers with the knowledge and skills to become independent users of this potent environment so that they can tackle any problem related to data wrangling and analytics, numerical computing, statistics, and machine learning.

### URL

https://deepr.gagolewski.com/index.html

### Suggested labels

Book, training

**Comment by pratikunterwegs** [comment]
Thanks for linking to this, looks like a nice resource! I went straight to the section on 'Interfacing Compiled Code' to see what was in it.

I liked the explanation of how to link to C/C++ without intermediary packages, but I would suggest folks use {Rcpp} for production code, as (I think) it helps avoid mistakes and offers plenty of syntactic sugar that makes life easier.
